### PR TITLE
Add command api to docs json

### PR DESCRIPTION
**Pull request by devinhalladay**
I'm not totally sure I'm doing this right, the docs setup is super confusing lol.

### CODE REVIEW

**Comment by lunaroyster** [comment]
let's jam on this today (monday)

File: modules/extensions/src/apis.json [hunk 1]
```diff
@@ -140,5 +140,22 @@
...
+      "CommandsFn",
+      "CreateCommand",
+      "Run",
+      "SerializableValue",
+      "BaseCommandArgs",
+      "ActionCommandArgs",
+      "ContextCommandArgs",
+      "CommandArgs",
+      "CommandProxy"
+    ]
```
Comment (lunaroyster) [inline modules/extensions/src/apis.json:159]: commandproxy is an implementation detail, removed it

```suggestion
    "typeNames": [
      "CommandFnArgs",
      "CommandsFn",
      "CreateCommand",
      "Run",
      "SerializableValue",
      "BaseCommandArgs",
      "ActionCommandArgs",
      "ContextCommandArgs",
      "CommandArgs",
    ]
```

**Review by lunaroyster — APPROVED** [review]
Looks good!

**Comment by lafkpages** [comment]
In these docs it says that the commands API is experimental but I could only get it working via `extensionPort.internal`.

**Comment by lunaroyster** [comment]
did you add the `experimental-api` scope?

**Comment by lunaroyster** [comment]
rebased this to pick up docs generation test in CI

**Pull request merged**